src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -272,25 +272,27 @@ export const VideoTileBase: React.FC<VideoTileBaseProps> = ({
             </TextContainer>
           </CSSTransition>
         </SwitchTransition>
-        <ContextMenu
-          placement="bottom-end"
-          items={
-            publisherMode
-              ? publisherKebabMenuItems
-              : [
-                  {
-                    icon: <SvgGlyphCopy />,
-                    onClick: onCopyVideoURLClick,
-                    title: 'Copy video URL',
-                  },
-                ]
-          }
-          trigger={
-            <KebabMenuButtonIcon onClick={() => null} variant="tertiary" size="small">
-              <SvgGlyphMore />
-            </KebabMenuButtonIcon>
-          }
-        />
+        {!isUploading && (
```

Comment:
I'm not sure fully hiding the trigger is the right way to go. This will make the available space change between uploading and regular tile and can cause some layout shifts. Instead we could add a prop to `KebabMenuButtonIcon` to make the button present in the layout but disabled